SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -93,6 +107,33 @@ void UGlobalStateManager::ApplyAcceptingPlayersUpdate(bool bAcceptingPlayersUpda
 	}
 }
 
+void UGlobalStateManager::SendShutdownMultiProcessRequest(bool bValue)
+{
+	Internal_SendShutdownMultiProcessRequest();
+}
+
+void UGlobalStateManager::Internal_SendShutdownMultiProcessRequest()
+{
+	FURL DummyURL;
```

Comment:
I think this would be a good spot to add some comments about this general logic and why this solution was settled upon.